docs/egress.md

```diff
@@ -9,6 +9,7 @@
   - [AppliedTo](#appliedto)
   - [EgressIP](#egressip)
   - [ExternalIPPool](#externalippool)
+  - [TrafficShaping](#trafficshaping)
```

Comment:
The other headers in this section use the field name, it should do the same here.

Reply:
Changed.